src/Aquifer.Public.API/Endpoints/Resources/Updates/List/Endpoint.cs

```diff
@@ -1,10 +1,12 @@
-﻿using Aquifer.Data;
+﻿using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
 using FastEndpoints;
 using Microsoft.EntityFrameworkCore;
 
 namespace Aquifer.Public.API.Endpoints.Resources.Updates.List;
 
-public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, Response>
+public class Endpoint(AquiferDbContext dbContext, ICachingLanguageService _cachingLanguageService) : Endpoint<Request, Response>
```

Comment:
Inconsistent naming with one thing having an underscore and the other not. I don't have a preference either way in these primary constructors, but we should at least be consistent per file.